src/radio-common/customs.ts

```diff
@@ -74,31 +73,24 @@ export default class RadioFilter {
     // Resolve signer to indexer identity via registry
     const isOperator = await this.isOperator(client, sender);
     //TODO: skipping for easier ease of testing on other indexers
-    const indexer = isOperator
-      ? (await this.isOperatorOf(client, sender))[0]
-      : sender;
+    const indexerAddress = await this.isOperatorOf(
+      client,
+      process.env.RADIO_OPERATOR
```

Comment:
Here we are resolving sender identity of the message, not so much our own radio operator, I think we should keep the input as `sender`?

Reply:
oh damn, yes! That slipped me 🤦🏻